kstreams/debezium-mongodb/Dockerfile

```diff
@@ -2,14 +2,19 @@ FROM debezium/connect:0.8
 ENV KAFKA_CONNECT_MONGODB_DIR=$KAFKA_CONNECT_PLUGINS_DIR/kafka-connect-mongodb
 
 USER root
-RUN yum -y install git maven && yum clean all
+RUN yum -y install git maven curl && yum clean all
+
+RUN cd /kafka/libs && \
+  curl -s https://oss.sonatype.org/content/repositories/releases/org/mongodb/mongodb-driver/3.8.2/mongodb-driver-3.8.2.jar --remote-name && \
+  curl -s https://oss.sonatype.org/content/repositories/releases/org/mongodb/mongodb-driver-core/3.8.2/mongodb-driver-core-3.8.2.jar --remote-name && \
+  curl -s https://oss.sonatype.org/content/repositories/releases/org/mongodb/bson/3.8.2/bson-3.8.2.jar --remote-name
 
 USER kafka
 
 # Deploy MongoDB Sink Connector
-RUN mkdir $KAFKA_CONNECT_MONGODB_DIR && cd $KAFKA_CONNECT_MONGODB_DIR && \
+RUN mkdir -p $KAFKA_CONNECT_MONGODB_DIR && cd $KAFKA_CONNECT_MONGODB_DIR && \
   git clone https://github.com/hpgrahsl/kafka-connect-mongodb.git && \
```

Comment:
I'd suggest to check out the v1.2.0 tag, to be on the safe side if the project's master branch moves to a new version down the road.

Reply:
Absolutely! Great call @gunnarmorling !